styles/settings-view.less

```diff
@@ -290,6 +290,10 @@
     margin: @section-padding 0;
   }
 
+  .sub-section.collapsed .package-container .row {
```

Comment:
Using `.sub-section.collapsed .package-container` makes sections with zero packages to change in size when their headers are clicked. Would that be ok?